src/olympia/constants/base.py

```diff
@@ -204,6 +204,7 @@
     'opencollective.com',
     'patreon.com',
     'paypal.com',
+    'buymeacoffee.com',
```

Comment:
It looks like this list of domains is currently in alphabetical order. Could you please move this to the top of the list?

Reply:
Sure @bobsilverberg will make that change ASAP!

Reply:
Thanks!